convert2rhel/pkghandler.py

```diff
@@ -158,10 +160,10 @@ def call_yum_cmd(
         repos_to_enable = system_info.get_enabled_rhel_repos()
 
     for repo in repos_to_enable:
-        cmd += " --enablerepo=%s" % repo
+        cmd.append("--enablerepo=%s" % repo)
 
     if args:
```

Comment:
I think this check is not entirely necessary here, at the beginning we are already saying that if the `args is None`, then `args = []`.

What do you think, @abadger?

Reply:
I agree!  Removing it now.